dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/relationship_field.jsp

```diff
@@ -93,6 +97,79 @@
 
 	// issue-19204
 	double randomNumber = Math.random();
+
+    final Map<String, String> specialFields = ImmutableMap.of(
+            "title", "Title",

```

Comment:
For these values we need to use 
`LanguageUtil.get(pageContext, "Title")`
`LanguageUtil.get(pageContext, "Title-Image")`
`LanguageUtil.get(pageContext, "Language")`

Reply:
Done.